api/app/controllers/spree/graphql/graphql_controller.rb

```diff
@@ -0,0 +1,56 @@
+module Spree
+  module Graphql
+    class GraphqlController < ::Doorkeeper::ApplicationMetalController
+      before_action :authorize_request, except: :login
+
+      def login
+        if (user = fetch_user)
+          render json: JwtToken.create(user), status: :ok
+        else
+          render json: { error: 'unauthorized' }, status: :unauthorized
+        end
+      end
+
+      def create
+        query_string = params[:query]
+        query_variables = ensure_hash(params[:variables])
+        result = GraphqlSchema.execute(query_string, variables: query_variables, context: { current_spree_user: @current_spree_user })
```

Comment:
```suggestion
        result = GraphqlSchema.execute(query_string, variables: query_variables, context: { current_spree_user: @current_spree_user })
```

We also need `current_order` context here